contracts/registrar/ZNSSubRegistrarTrunk.sol

```diff
@@ -91,57 +92,48 @@ contract ZNSSubRegistrar is AAccessControlled, ARegistryWired, UUPSUpgradeable,
         DistributionConfig calldata distrConfig,
         PaymentConfig calldata paymentConfig
     ) external override returns (bytes32) {
+        return _coreSubdomainRegister(
```

Comment:
Here and in other places it would be nicer if the internal `_core` functions were consistent with the naming of the public callers as much as they can be, so `_coreRegisterSubdomain`, this is clearer as well as the name describes the action the function is taking